File: Coder Revolt - Web Browser Open Source/Coder Revolt - Web Browser Open Source/Bookmarks.vb
Public Class Bookmarks
    Private Sub ListBox2_SelectedIndexChanged(sender As System.Object, e As System.EventArgs) Handles ListBox1.SelectedIndexChanged
        ListBox2.SelectedIndex = ListBox1.SelectedIndex
    End Sub

    Private Sub Bookmarks_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load
        Me.TopMost = True

        Timer1.Start()
    End Sub

    Private Sub Button3_Click(sender As System.Object, e As System.EventArgs) Handles Button3.Click
        Try
            ListBox1.Items.Remove(ListBox1.SelectedItem)
            ListBox2.Items.Remove(ListBox2.SelectedItem)
            MessageBox.Show("Deleted!")
            Button1.PerformClick()
        Catch ex As Exception



        End Try

    End Sub

    Private Sub Button1_Click(sender As System.Object, e As System.EventArgs) Handles Button1.Click
        Dim w As IO.StreamWriter

        Try
            Dim i As Integer
            w = New IO.StreamWriter("C:\CoderRevolt\WebBrowser\V1\Bookmarks.gwbb")
            For i = 0 To ListBox1.Items.Count - 1
                w.WriteLine(ListBox1.Items.Item(i))
            Next
            w.Close()
        Catch ex As Exception

        End Try
        Try
            Dim i As Integer
            w = New IO.StreamWriter("C:\CoderRevolt\WebBrowser\V1\Bookmarks2.gwbb")
            For i = 0 To ListBox2.Items.Count - 1
                w.WriteLine(ListBox2.Items.Item(i))
            Next
            w.Close()
        Catch ex As Exception

        End Try
    End Sub

    Private Sub Timer1_Tick(sender As System.Object, e As System.EventArgs) Handles Timer1.Tick


    End Sub

    Private Sub Button2_Click(sender As System.Object, e As System.EventArgs) Handles Button2.Click
        Try

            Main.RadioButton2.Checked = True

            Main.ComboBox1.Text = ListBox2.SelectedItem.ToString
            Main.Button6.PerformClick()

        Catch ex As Exception

        End Try

    End Sub



    Private Sub Button4_Click_1(sender As Object, e As EventArgs) Handles Button4.Click
        Me.Hide()
        Button1.PerformClick()
    End Sub
End Class

File: C# Converted Code V.1/Bookmarks.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
public class Bookmarks
{
	private void ListBox2_SelectedIndexChanged(System.Object sender, System.EventArgs e)
	{
		ListBox2.SelectedIndex = ListBox1.SelectedIndex;
	}

	private void Bookmarks_Load(System.Object sender, System.EventArgs e)
	{
		this.TopMost = true;

		Timer1.Start();
	}

	private void Button3_Click(System.Object sender, System.EventArgs e)
	{
		try {
			ListBox1.Items.Remove(ListBox1.SelectedItem);
			ListBox2.Items.Remove(ListBox2.SelectedItem);
			MessageBox.Show("Deleted!");
			Button1.PerformClick();

		} catch (Exception ex) {


		}

	}

	private void Button1_Click(System.Object sender, System.EventArgs e)
	{
		IO.StreamWriter w = default(IO.StreamWriter);

		try {
			int i = 0;
			w = new IO.StreamWriter("C:\\CoderRevolt\\WebBrowser\\V1\\Bookmarks.gwbb");
			for (i = 0; i <= ListBox1.Items.Count - 1; i++) {
				w.WriteLine(ListBox1.Items.Item(i));
			}
			w.Close();

		} catch (Exception ex) {
		}
		try {
			int i = 0;
			w = new IO.StreamWriter("C:\\CoderRevolt\\WebBrowser\\V1\\Bookmarks2.gwbb");
			for (i = 0; i <= ListBox2.Items.Count - 1; i++) {
				w.WriteLine(ListBox2.Items.Item(i));
			}
			w.Close();

		} catch (Exception ex) {
		}
	}


	private void Timer1_Tick(System.Object sender, System.EventArgs e)
	{

	}

	private void Button2_Click(System.Object sender, System.EventArgs e)
	{

		try {
			Main.RadioButton2.Checked = true;

			Main.ComboBox1.Text = ListBox2.SelectedItem.ToString;
			Main.Button6.PerformClick();


		} catch (Exception ex) {
		}

	}



	private void Button4_Click_1(object sender, EventArgs e)
	{
		this.Hide();
		Button1.PerformClick();
	}
	public Bookmarks()
	{
		Load += Bookmarks_Load;
	}
}
